### PR TITLE
Added RssItem and RssFeed classes

### PR DESCRIPTION
**Pull request by Exploding-Potato**
I think it's pretty self-explanatory. Just check whether I've fucked something up or not.
Also, tests.

### CODE REVIEW

**Comment by Exploding-Potato** [comment]
Maybe I should've used a HashSet instead od an ArrayList, your choice if it'll stay.

**Comment by Exploding-Potato** [comment]
Or maybe a LinkedHashSet

File: src/main/java/rssfeed/RssFeed.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+import java.util.Objects;
+
+public class RssFeed {
+    public String title;
+    public String description;
+    public String link;
+    public String language;
+    public String copyright;
+
+    public ArrayList<RssItem> items = new ArrayList<RssItem>();
```
Comment (Kempior) [inline src/main/java/rssfeed/RssFeed.java]:
```suggestion
    public List<RssItem> items = new ArrayList<>();
```
Prefer interfaces over concrete implementations

File: src/main/java/rssfeed/RssFeed.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if (o == null || getClass() != o.getClass()) return false;
```
Comment (Kempior) [inline src/main/java/rssfeed/RssFeed.java]:
```suggestion
        if (!(o instanceof RssFeed)) return false;
```

File: src/main/java/rssfeed/RssItem.java [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+        imgUrl = other.imgUrl;
+        pubDate.setTime(other.pubDate.getTime());
+
+        guid = other.guid;
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if (o == null || getClass() != o.getClass() || this.hashCode() != o.hashCode()) return false;
```
Comment (Kempior) [inline src/main/java/rssfeed/RssItem.java]:
```suggestion
        if (!(o instanceof RssItem)) return false;
```
"It is not required that if two objects are unequal according to the equals(java.lang.Object) method, then calling the hashCode method on each of the two objects must produce distinct integer results."

**Comment by Exploding-Potato** [comment]
`Objects.equals(var, other.var)` was used by IntelliJ automatic equals() and hashCode() generation.
Is there a reason `var.equals(other.var)` is preferred?

**Pull request merged**